src/Stripe.net/Entities/Issuing/Transactions/TransactionPurchaseDetailsFuel.cs

```diff
@@ -22,7 +22,7 @@ public class TransactionPurchaseDetailsFuel : StripeEntity<TransactionPurchaseDe
         /// decimal places.
         /// </summary>
         [JsonProperty("unit_cost_decimal")]
-        public decimal? UnitCostDecimal { get; set; }
+        public decimal UnitCostDecimal { get; set; }
```

Comment:
Huh I'm surprised by that one, it's recent and I thought I reviewed the spec carefully. Do you know if it changed since we built it?